src/cli/embedded_help.rb

```diff
@@ -54,9 +54,10 @@ def self.preprocess_ruby_script(s)
 module Kernel
   # ":" is our root path to the embedded file system
   # make sure it is in the ruby load path
-  if ENV['RUBYLIB']
-    ENV['RUBYLIB'].split(File::PATH_SEPARATOR).each {|lib| $LOAD_PATH.unshift(lib)}
-  end
+  # TJC remove RUBYLIB env and use --include or -I via cli args instead
+  #if ENV['RUBYLIB']
+  #  ENV['RUBYLIB'].split(File::PATH_SEPARATOR).each {|lib| $LOAD_PATH.unshift(lib)}
+  #end
```

Comment:
I'm all good with these changes, but I think we need to document that properly **somewhere**, don't we? We should change the wording in https://nrel.github.io/OpenStudio-user-documentation/reference/command_line_interface/#environment-variables

Reply:
Yes, I'll take care of the documentation before I merge in this PR. We need to remove some of the wording in those docs related to using other ruby ENVs as well so I'll clean that up.

Reply:
docs are updated to reflect this change